.circleci/config.yml

```diff
@@ -266,17 +278,21 @@ jobs:
       image: << parameters.machine_image >>
     resource_class: << parameters.resource_class >>
     working_directory: /tmp/workspace/repo
+    parallelism: 2
     steps:
       - attach_workspace:
           at: /tmp/workspace
       - run:
           name: Load docker localstack-full image
           command: docker load -i target/localstack-docker-image-<< parameters.platform >>.tar
       - run:
-          name: Run tests
+          name: Run integration tests
+          # circleci split returns newline separated list, so `tr` is necessary to prevent problems in the Makefile
           command: |
-            PYTEST_ARGS='--junitxml=target/reports/test-report-<< parameters.platform >>.xml' \
-            COVERAGE_FILE='target/coverage/.coverage.<< parameters.platform >>' \
+            TEST_FILES=$(circleci tests glob "tests/integration/**/test_*.py" | circleci tests split --split-by=timings | tr '\n' ' ')
+            PYTEST_ARGS="-o junit_family=legacy --junitxml=target/reports/test-report-<< parameters.platform >>-${CIRCLE_NODE_INDEX}.xml" \
```

Comment:
junit_family=legacy is necessary for filenames to be included in the resulting junit-style xml.
This is necessary for the timing split to work.